src/api/app/components/diff_component.html.haml

```diff
@@ -5,7 +5,25 @@
       .line-container{ id: }
         %div{ class: "line #{line.state}" }>
           - case line.state
-          - when 'comment', 'range'
+          - when 'range'
+            - if source_file
+              = link_to(source_file) do
+                .number<
+                  %i.fas.fa-arrows-up-to-line
```

Comment:
I understand what you want to imply here but I don't think this is clear. You will have to get the reference to PRs which many people will not have.

Reply:
Yeah I think those are way better communicating what they do...